app/src/main/java/com/hedvig/app/ApplicationModule.kt

```diff
@@ -419,9 +422,31 @@ val marketingModule = module {
 }
 
 val offerModule = module {
-    viewModel<OfferViewModel> { (ids: List<String>, shouldShowOnNextAppStart: Boolean) ->
-        OfferViewModelImpl(ids, get(), get(), get(), get(), get(), get(), get(), shouldShowOnNextAppStart, get(), get())
+    viewModel<OfferViewModel> { parametersHolder: ParametersHolder ->
+        val (
+            ids: List<String>, shouldShowOnNextAppStart: Boolean, insurelyDataCollectionReferenceUuid: String?,
```

Comment:
Only `insurelyDataCollectionReferenceUuid` changed here, but added named args because it was hard to reason about the order